packages/ai-jsx/src/lib/openai.tsx

```diff
@@ -248,13 +248,13 @@ export async function* OpenAICompletionModel(
 
   for await (const event of openAiEventsToJson<CreateCompletionResponse>(responseIterator)) {
     logger.trace({ event }, 'Got createCompletion event');
+    yield event.choices[0].text;
```

Comment:
It feels nice to remove the `+=` responsibility from this component. And it feels good for components to be able to explicitly say they'll be `appendOnly`, in cases where that can be a simplifying assumption.

Reply:
Yeah those were always a little icky. Now the ickiness is one layer closer to being removed :)